README.md

```diff
@@ -116,9 +108,10 @@ The `GITHUB_AUTH_TOKEN` has to be set to a valid [token](#Authentication)
 docker run -e GITHUB_AUTH_TOKEN=token gcr.io/openssf/scorecard:stable --show-details --repo=https://github.com/ossf/scorecard
 ```
 
-### Using repository URL
+### Basic Usage
+#### Using repository URL
 
-The program can run using just one argument, the URL of the repo:
+Scorecards can run using just one argument, the URL of the target repo:
 
 ```shell
 $ go get github.com/ossf/scorecard/v2
```

Comment:
This has to be `go install github.com/ossf/scorecard/v2@latest`

Reply:
Thank you; updated!